### PR TITLE
DIGI_9_ListApplies

### PR DESCRIPTION
**Pull request by kvindhya**
1. Created AppliesController for get method
2. Created View with method name Index
3. Modified Enpoints
4. Created AppliesModel
5. Created AppliesLibrary
6. Modified Library Utility

### CODE REVIEW

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AppliesController.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+namespace DigiOrderEnterprise.Controllers
+{
+    public class AppliesController : Controller
+    {
+        AppliesLibrary appliesLibrary = null;
+        // GET: Applies
+
+        public ActionResult Index()
+
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AppliesController.cs:19]: Remove line 19

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AppliesController.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        {
+            List<AppliesModel> modelList = null;
+
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AppliesController.cs:22]: Remove line 22

File: DigiOrderEnterprise/DigiOrderEnterprise/Views/Applies/Index.cshtml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+﻿@model List<DigiOrderEnterpriseModel.AppliesModel>
+@{
+    ViewBag.Title = "Index";
+    Layout = "~/Views/Shared/_EnterpriseLayout.cshtml";
+}
+
+<div style="color:white;background-color:red; text-align:center">@ViewBag.ErrorMessage</div>
+<div style="color:white;background-color:green; text-align:center">@TempData["Success"]</div>
+
+
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Views/Applies/Index.cshtml:10]: Remove line 6,9 and 10

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AppliesController.cs [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+using System.Threading.Tasks;
+using System.Web.Mvc;
+
+
+namespace DigiOrderEnterprise.Controllers
+{
+    public class AppliesController : Controller
+    {
+        AppliesLibrary appliesLibrary = null;
+        // GET: Applies
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AppliesController.cs:16]: Remove line 16

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AppliesController.cs [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+            try
+            {
+                appliesLibrary = new AppliesLibrary();
+                Task<HttpResponseMessage> response = appliesLibrary.getAllApplies();
+                string data = response.Result.Content.ReadAsStringAsync().Result;
+                modelList = JsonConvert.DeserializeObject<List<AppliesModel>>(data);
+            }
+            catch (Exception ex)
+            {
+                ViewBag.ErrorMessage = "Data is Not found from API";
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AppliesController.cs:30]: change message "Something went wrong.Contact administrator.."

File: DigiOrderEnterprise/DigiOrderEnterprise/Views/Applies/Index.cshtml [hunk 6]
```diff
@@ -0,0 +1,56 @@
+﻿@model List<DigiOrderEnterpriseModel.AppliesModel>
+@{
+    ViewBag.Title = "Index";
+    Layout = "~/Views/Shared/_EnterpriseLayout.cshtml";
+}
+<div style="color:white;background-color:red; text-align:center">@ViewBag.ErrorMessage</div>
+<div style="color:white;background-color:green; text-align:center">@TempData["Success"]</div>
+@{ if (Model != null)
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Views/Applies/Index.cshtml:8]: Check count >0

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/EndPoint.cs [hunk 7]
```diff
@@ -1,14 +1,9 @@
...
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
+﻿
 namespace DigiOrderEnterpriseLibrary
 {
-    class EndPoint
+    public class Endpoints
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterpriseLibrary/EndPoint.cs:4]: Remove public

File: DigiOrderEnterprise/DigiOrderEnterpriseModel/AppliesModel.cs [hunk 8]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterpriseModel/AppliesModel.cs:7]: Add extra line

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/DigiOrderEnterpriseLibrary.csproj [hunk 9]
```diff
@@ -31,7 +31,8 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="DigiOrderEnterpriseModel">
+    <Reference Include="DigiOrderEnterpriseModel, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
```
Comment (RAMUNIMMALA) [inline DigiOrderEnterprise/DigiOrderEnterpriseLibrary/DigiOrderEnterpriseLibrary.csproj:35]: Remove these lines and keep the older

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/EndPoint.cs [hunk 10]
```diff
@@ -3,7 +3,7 @@ namespace DigiOrderEnterpriseLibrary
 {
     class EndPoint
     {
-        public const string ALL_CATEGORIES = "Category";
```
Comment (RAMUNIMMALA) [inline DigiOrderEnterprise/DigiOrderEnterpriseLibrary/EndPoint.cs:6]: Reason to remove the "ALL_CATEGORIES"

File: DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj [hunk 11]
```diff
@@ -43,9 +43,10 @@
...
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="AppliesModel.cs" />
     <Compile Include="CategoryModel.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="TaxPageModel.cs" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-</Project>
+</Project>
```
Comment (RAMUNIMMALA) [inline DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj:52]: No EOF is available

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/DigiOrderEnterpriseLibrary.csproj [hunk 12]
```diff
@@ -58,4 +59,4 @@
     <None Include="packages.config" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-</Project>
\ No newline at end of file
+</Project>
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterpriseLibrary/DigiOrderEnterpriseLibrary.csproj:62]: add EOF

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/DigiOrderEnterpriseLibrary.csproj [hunk 13]
```diff
@@ -31,7 +31,7 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="DigiOrderEnterpriseModel">
+   <Reference Include="DigiOrderEnterpriseModel">
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterpriseLibrary/DigiOrderEnterpriseLibrary.csproj:34]: Allign in order

File: DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj [hunk 14]
```diff
@@ -116,10 +116,12 @@
     <Compile Include="App_Start\BundleConfig.cs" />
     <Compile Include="App_Start\FilterConfig.cs" />
     <Compile Include="App_Start\RouteConfig.cs" />
+    <Compile Include="Controllers\AppliesController.cs" />
     <Compile Include="Controllers\CategoryController.cs" />
     <Compile Include="Controllers\DashboardController.cs" />
     <Compile Include="Controllers\HomeController.cs" />
     <Compile Include="Controllers\MenuItemController.cs" />
+    <Compile Include="Controllers\StoreController.cs" />
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj:124]: remove store controller in .csproj

File: DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj [hunk 15]
```diff
@@ -250,6 +252,7 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="App_Data\" />
+    <Folder Include="Data\" />
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj:255]: Remove line 255

File: DigiOrderEnterprise/DigiOrderEnterprise/Views/Store/Index.cshtml [hunk 16]
```diff
@@ -53,8 +53,8 @@
                         </td>
                         <td class="invert">
                             <div class="rem">
-                                <div class="entry value">EDIT</div>
-                                <div class="entry value"> DELETE </div>
+                                 <div class="entry value">EDIT</div>
+                                 <div class="entry value"> DELETE </div>
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Views/Store/Index.cshtml:57]: Remove spaces before div tags

File: DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj [hunk 17]
```diff
@@ -249,7 +250,7 @@
     <Content Include="Views\Shared\_EnterpriseLayout.cshtml" />
   </ItemGroup>
   <ItemGroup>
-    <Folder Include="App_Data\" />
+    <Folder Include="App_Data\" />
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj:253]: Remove spaces after > tag

File: DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj [hunk 18]
```diff
@@ -116,10 +116,11 @@
     <Compile Include="App_Start\BundleConfig.cs" />
     <Compile Include="App_Start\FilterConfig.cs" />
     <Compile Include="App_Start\RouteConfig.cs" />
+    <Compile Include="Controllers\AppliesController.cs" />
     <Compile Include="Controllers\CategoryController.cs" />
     <Compile Include="Controllers\DashboardController.cs" />
     <Compile Include="Controllers\HomeController.cs" />
-    <Compile Include="Controllers\MenuItemController.cs" />
+    <Compile Include="Controllers\MenuItemController.cs" />
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/DigiOrderEnterprise.csproj:123]: Remove spaces after > tag

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/EndPoint.cs (hunk 10 above)
Comment (kvindhya) [reply DigiOrderEnterprise/DigiOrderEnterpriseLibrary/EndPoint.cs:6]: resolved as per comments

File: DigiOrderEnterprise/DigiOrderEnterpriseModel/AppliesModel.cs (hunk 8 above)
Comment (kvindhya) [reply DigiOrderEnterprise/DigiOrderEnterpriseModel/AppliesModel.cs:4]: resolved

**Comment by RAMUNIMMALA** [comment]
code review completed

**Pull request merged**